ci/azure-rustfmt.yml

```diff
@@ -13,5 +13,5 @@ jobs:
         cargo fmt --version
       displayName: Install rustfmt
     - script: |
-        cargo fmt --all -- --check
+        rustfmt --check --edition 2018 $(find . -name '*.rs' -print)
```

Comment:
The hypothesis is that `cargo fmt` does not support feature flags and there is no way to specify the feature flags to enable.

Reply:
Can we add a comment inline explaining that?